src/main/java/com/checkmarx/flow/CxFlowRunner.java

```diff
@@ -262,15 +264,24 @@ private void commandLineRunner(ApplicationArguments args) throws ExitThrowable {
                 repoType = ScanRequest.Repository.GITHUB;
 
                 if(ScanUtils.empty(namespace) ||ScanUtils.empty(repoName)||ScanUtils.empty(mergeId)){
-                    log.error("Namespace/Repo/MergeId must be provided for GITHUBPULL bug tracking");
+                    log.error("--namespace, --repo and --merge-id must be provided for GITHUBPULL bug tracking");
                     exit(1);
                 }
                 mergeNoteUri = gitHubProperties.getMergeNoteUri(namespace, repoName, mergeId);
                 break;
             case GITLABMERGE:
             case gitlabmerge:
-                log.info("GitLab Merge not currently supported from command line");
-                exit(1);
+                bugType = BugTracker.Type.GITLABMERGE;
+                bt = BugTracker.builder()
+                        .type(bugType)
+                        .build();
+                repoType = ScanRequest.Repository.GITLAB;
+
+                if(ScanUtils.empty(projectId)||ScanUtils.empty(mergeId)){
+                    log.error("--project-id and --merge-id must be provided for GITLABMERGE bug tracking");
```

Comment:
Using template formatting would probably be better:
`log.error("--project-id and --merge-id must be provided for {} bug tracking", bugType)`